scripts/DocGenerator.php

```diff
@@ -22,7 +22,7 @@
 use phpDocumentor\Reflection\FileReflector;
 
 /**
- * Parses given files and builds JSON documentation.
+ * Parses given files and builds documentation for our common docs site.
```

Comment:
There is, I'm working on updating it right now.

http://dwsupplee.github.io/gcloud-php/#/